src/main/java/com/example/everkan/email/EmailService.java

```diff
@@ -0,0 +1,44 @@
+package com.example.everkan.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Service
+public class EmailService implements EmailSender {
+
+    private final static Logger LOGGER = LoggerFactory
+            .getLogger(EmailService.class);
+    private final String from;
+    private final JavaMailSender mailSender;
+
+    public EmailService(@Value("${app.email_address}") String from, JavaMailSender mailSender) {
```

Comment:
The `@Value` annotation can bypass validation.

To avoid that, you can read it from a `@ConfigurationProperties` object and create the instance in a `@Bean` method of a `@Configuration`.

Reply:
Fixed